<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientPicker</class>
 <widget class="QWidget" name="GradientPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gradient" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
  </widget>
  <widget class="QWidget" name="slider" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 241, 241);</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="box">
     <item>
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Линейный </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Радиальный</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="minus_marker">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>125</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus_marker">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>125</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mrk1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>130</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>39</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mrk2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>39</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="dialLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
